.github/workflows/build-wheel-macos-arm64.yaml

```diff
@@ -403,7 +403,7 @@ jobs:
     - name: Install OQC client
       if: ${{ matrix.python_version == '3.9' || matrix.python_version == '3.10'}}
       run: |
-        python${{ matrix.python_version }} -m pip install oqc-qcaas-client
+        python${{ matrix.python_version }} -m pip install 'oqc-qcaas-client<3.6.0'
```

Comment:
Could you add a comment in the python OQC device about this limitation?

Reply:
Done

Reply:
I meant in the device documentation, `Catalyst/catalyst/frontend/catalyst/third_party/oqc/oqc_device.py` otherwise the users cannot know what to install.